docs/source/topics/calculations/usage.rst

```diff
@@ -312,38 +312,135 @@ Note that the source path can point to a directory, in which case its contents w
-The simplest is retrieving a single file, whose filename you know before hand and you simply want to copy with the same name in the retrieved folder.
-Imagine you want to retrieve the files ``output1.out`` and ``output_folder/output2.out`` you would simply add them as strings to the retrieve list:
-
-.. code:: python
-
-    calc_info.retrieve_list = ['output1.out', 'output_folder/output2.out']
-
-The retrieved files will be copied over keeping the exact names and hierarchy.
-If you require more control over the hierarchy and nesting, you can use tuples of length three instead, with the following items:
+The retrieve list is a list of relative filepaths to define what files and folders should be retrieved.
+By default, any subfolders in the remote working directory will be discarded when the file or folder is copied to the retrieved folder and only the filename is kept.
+If the file hierarchy should be respected, a tuple of length three can be added to the retrieve list, with the following format:
 
     * `source relative path`: the relative path, with respect to the working directory on the remote, of the file or directory to retrieve
     * `target relative path`: the relative path where to copy the files locally in the retrieved folder. The string `'.'` indicates the top level in the retrieved folder.
     * `depth`: the number of levels of nesting in the folder hierarchy to maintain when copying, starting from the deepest file
 
-For example, imagine the calculation will have written a file in the remote working directory with the folder hierarchy ``some/remote/path/files/output.dat``.
-If you want to copy the file, with the final resulting path ``path/files/output.dat``, you would specify:
+To illustrate the various possibilities, consider the following example file hierarchy in the remote working directory:
 
-.. code:: python
+.. code:: bash
 
-    calc_info.retrieve_list = [('some/remote/path/files/output.dat', '.', 2)]
+    ├─ path
+    |  ├── sub
+    │  │   ├─ file_c.txt
+    │  │   └─ file_d.txt
+    |  └─ file_b.txt
+    └─ file_a.txt
 
-The depth of two, ensures that only two levels of nesting are copied.
-If the output files have dynamic names that one cannot know beforehand, the ``'*'`` glob pattern can be used.
-For example, if the code will generate a number of XML files in the folder ``relative/path/output`` with filenames that follow the pattern ``file_*[0-9].xml``, you can instruct to retrieve all of them as follows:
+Below, you will find examples for various use cases of files and folders to be retrieved.
+They will first show the format of the ``retrieve_list``, followed by a schematic depiction of the final file hierarchy that is created in the retrieved folder.
 
-.. code:: python
+Explicit file or folder
+.......................
+
+.. code:: bash
+
+    retrieve_list = ['file_a.txt']
+
+    └─ file_a.txt
+
+.. code:: bash
+
+    retrieve_list = ['path']
```

Comment:
This should be most probably:
retrieve_list = [ ('sub/file_c.txt','.',2), ('sub/file_d.txt','.',2), ('file_b.txt','.',1) ]